tests/phpunit/Checker/Checks/Plugin_Readme_Check_Tests.php

```diff
@@ -154,17 +159,17 @@ public function test_run_md_with_errors() {
 		$this->assertArrayHasKey( 'code', $warnings['readme.md'][0][0][1] );
 		$this->assertEquals( 'invalid_license', $warnings['readme.md'][0][0][1]['code'] );
 
-		// Check for trunk stable tag warning.
-		$this->assertArrayHasKey( 0, $warnings['readme.md'] );
-		$this->assertArrayHasKey( 0, $warnings['readme.md'][0] );
-		$this->assertArrayHasKey( 'code', $warnings['readme.md'][0][0][2] );
-		$this->assertEquals( 'trunk_stable_tag', $warnings['readme.md'][0][0][2]['code'] );
-
-		// Check for stable tag mismatch file warning.
-		$this->assertArrayHasKey( 0, $warnings['readme.md'] );
-		$this->assertArrayHasKey( 0, $warnings['readme.md'][0] );
-		$this->assertArrayHasKey( 'code', $warnings['readme.md'][0][0][3] );
-		$this->assertEquals( 'stable_tag_mismatch', $warnings['readme.md'][0][0][3]['code'] );
+		// Check for trunk stable tag error.
+		$this->assertArrayHasKey( 0, $errors['readme.md'] );
+		$this->assertArrayHasKey( 0, $errors['readme.md'][0] );
+		$this->assertArrayHasKey( 'code', $errors['readme.md'][0][0][0] );
+		$this->assertEquals( 'trunk_stable_tag', $errors['readme.md'][0][0][0]['code'] );
+
+		// Check for stable tag mismatch file error.
+		$this->assertArrayHasKey( 0, $errors['readme.md'] );
+		$this->assertArrayHasKey( 0, $errors['readme.md'][0] );
```

Comment:
The exact same assertions are already done above, so these are redundant.

Reply:
Thanks @swissspidy for review. Please check attached screenshot to know how we get the error/notice data.

<img width="956" alt="Screenshot 2023-09-25 at 2 56 31 PM" src="https://github.com/10up/plugin-check/assets/10103365/d527d6d1-a7ba-4e38-8b6b-fab721eb75e8">

Reply:
That's not what I am referring to.

These two lines are literally duplicated:

```php
$this->assertArrayHasKey( 0, $errors['readme.md'] );
$this->assertArrayHasKey( 0, $errors['readme.md'][0] );
```

Lines 162-163 and 169-170

Reply:
```suggestion
```

Reply:
Ohh gotcha. Updated in https://github.com/10up/plugin-check/pull/279/commits/c723ec824f5e1130e0646f79e1c87ae9789d85ff